account_reconcile_oca/models/account_partial_reconcile.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 FactorLibre - Aritz Olea
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class AccountPartialReconcile(models.Model):
+
+    _inherit = "account.partial.reconcile"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        if self.env.context.get("no_foreign_currency", False):
+            for vals in vals_list:
+                vals["debit_amount_currency"] = 0
```

Comment:
Why not credit? It seems something weird to do it with this context. I would recommend to give some more context to your issue, how it was raised and so on...